### PR TITLE
SC-2287: Expose ES and RMQ ports + kibana service

### PR DESCRIPTION
**Pull request by alexanderM91**
#### Overview

- Ticket: https://spryker.atlassian.net/browse/SC-2287
- Suite-Nonsplit PR: https://github.com/spryker/suite-nonsplit/pull/3105
- Release Group: https://release.spryker.com/release-groups/view/2053

##### Improvements

- Introduced a new Kibana service for Docker Environment.
- Introduced a new TCP ElasticSearch server configuration.
- Introduced a new TCP RabbitMq server configuration.

### CODE REVIEW

File: generator/src/templates/service/kibana.yml.twig [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+         - private
+     healthcheck:
+         test: curl --cacert /usr/share/elasticsearch/config/certs/ca/ca.crt -s https://localhost:5601 >/dev/null; if [[ $$? == 52 ]]; then echo 0; else echo 1; fi
+         interval: 30s
+         timeout: 10s
+         retries: 5
+     environment:
+         - ELASTICSEARCH_URL=http://search:9200
+         - NODE_OPTIONS=--max-old-space-size=5000
+     depends_on:
```
Comment (sprymiker) [inline generator/src/templates/service/kibana.yml.twig]: Add 
```
command: "./bin/kibana --x-pack.enabled=false"
```

File: images/kibana/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# syntax = docker/dockerfile:experimental
```
Comment (sprymiker) [inline images/kibana/Dockerfile]: Please check if `command` does works

File: context/kibana/kibana.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Kibana is served by a back end server. This setting specifies the port to use.
```
Comment (sprymiker) [inline context/kibana/kibana.yml]: Index: *

Comment (alexanderM91) [reply context/kibana/kibana.yml]: Hi, @sprymiker, default Kibana configuration doesn't have an option to set `default index pattern`.
I found the way how to set it https://github.com/wazuh/wazuh-kibana-app/wiki/Set-default-index-pattern-in-Kibana but I sure it's not acceptable )))

Kibana default configuration settings you can find here: https://www.elastic.co/guide/en/kibana/5.6/settings.html.

https://www.elastic.co/guide/en/kibana/5.6/advanced-options.html

Comment (alexanderM91) [reply context/kibana/kibana.yml]: We can update the default index pattern: https://www.elastic.co/guide/en/kibana/5.6/advanced-options.html

**Pull request merged**